_lexicon/evidence.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: post
+title: Evidence
+description: Proof of a __Model__'s accuracy. An __Observation__ can be used as __Evidence__ for a __Model__ based on the correlation between the __Model__ and the __Observation__.
```

Comment:
I think the term "Proof" here has undesirable connotations.

Reply:
"Indication"? or "Suggestion" of a __Model__'s Accuracy?

Reply:
Both of those work. I have a slight preference for "Indication"